src/utils/runtime/src/lib.rs

```diff
@@ -114,9 +115,24 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
         fmt_layer.with_filter(filter)
     };
 
-    if settings.enable_jaeger_tracing {
-        todo!("jaeger tracing is not supported for now, and it will be replaced with minitrace jaeger tracing. Tracking issue: https://github.com/risingwavelabs/risingwave/issues/4120");
-    }
+    let tracing_layer = if settings.enable_jaeger_tracing {
```

Comment:
Would you please view history of this file and add exactly what it was here? There're many problems with the "example" usage of opentelemetry_jaeger.